services/port_manager/src/main/java/com/futurewei/alcor/portmanager/proxy/IpManagerProxy.java

```diff
@@ -104,19 +118,37 @@ private void addIpAddrRollback(AbstractIpAddrRollback rollback, IpAddrRequest ip
         addIpAddrRollback(new AllocateIpAddrRollback(ipManagerRestClient), ipv4Addr);
 
         //Allocate a random ipv6 address
```

Comment:
Good call to comment out IPv6 allocation for now. Let us revisit IPv6 allocation when other components are more ready. Created a tracking issue in case we forget. 

Issue https://github.com/futurewei-cloud/alcor/issues/227